static/app/components/events/interfaces/sourceMapsDebuggerModal.tsx

```diff
@@ -621,6 +622,148 @@ function InstalledSdkChecklistItem({
   );
 }
 
+function getToolUsedToUploadSourceMaps({
+  releaseUserAgent,
+}: Pick<FrameSourceMapDebuggerData, 'releaseUserAgent'>) {
+  return {
+    sentryCli: releaseUserAgent?.includes('sentry-cli'),
+    vitePlugin: releaseUserAgent?.includes('vite-plugin'),
+    webpackPlugin: releaseUserAgent?.includes('webpack-plugin'),
+    rollupPlugin: releaseUserAgent?.includes('rollup-plugin'),
+    esbuildPlugin: releaseUserAgent?.includes('esbuild-plugin'),
+  };
```

Comment:
are there going to be multiple user agents in the string or only one?

Reply:
only one! I have updated the code ^^